README.md

```diff
@@ -50,12 +51,16 @@ Web app to track the progress for Frogtoberfest. 🎃 🐸
 
 ### [just-handlebars-helpers](https://github.com/leapfrogtechnology/just-handlebars-helpers)
 
-A lightweight package with common Handlebars helpers. 
+A lightweight package with common Handlebars helpers.
 
 ### [Nepali-Date-Picker](https://github.com/leapfrogtechnology/Nepali-Date-Picker)
 
 Nepali date picker JavaScript plugin.
 
+### [oss-leaderboard](https://github.com/leapfrogtechnology/oss-leaderboard)
+
+OSS Leaderboard shows the top contributors based on their contribution activities on GitHub.
```

Comment:
```suggestion
OSS Leaderboard shows the top contributors based on their recent opensource contributions.
```

Reply:
There's a feature to get contributions based on time range,  so "recent" might not be a feasible word here. 

So the sentence formation can be something like:

> OSS Leaderboard shows the top contributors based on their opensource contributions. 

@kabirbaidhya

Reply:
OK.